tools/bbc/bbc.cpp

```diff
@@ -115,11 +115,11 @@ struct DriverOptions {
   bool dumpUnparseWithSymbols{false};
   bool dumpParseTree{false};
   bool dumpSymbols{false};
-  bool debugLinearFIR{false};
   bool debugResolveNames{false};
   bool debugSemantics{false};
   bool measureTree{false};
   bool runBackend{true};
+  bool dumpPreFIR{false};
```

Comment:
An alternative is to use the "llvm::cl" stuff for command-line handling.  It's already wired up in `bbc` and used in a few places. Grep for diable-burnside-todo.

Reply:
`bbc` was a clone of `f18` (obviously) with a bunch of stuff pitched overboard and some new things grafted on. So it's a Frankenstein.